content/zh-cn/_index.html

```diff
@@ -34,7 +34,7 @@
 {{% blocks/feature image="blocks" %}}
 
 <!-- #### Never Outgrow -->
-#### 处处适用
+#### 永不过时
```

Comment:
`Never Outgrow` 翻成 `永不过时`后，这个标题和正文有点 gap，可能需要再揣摩一下：

```
#### 永不过时

无论是本地测试，还是跨国公司，Kubernetes 的灵活性都能让你在应对复杂系统时得心应手。
```

Reply:
The commit you referred to was made 4 years ago. We don't need to stick to the opinions of previous translators; instead, let's discuss why we think one word is more suitable than another, as it would be easier.

I personally prefer "永不過時" over "處處適用". Could you provide some opinions on why "處處適用" might be a better choice?

Reply:
Oh, I got confused too.

The changes seem right (even though the corresponding description is weird), thanks!